wolox-books/src/services/bookService.js

```diff
@@ -0,0 +1,7 @@
+import booksApi from '../config/api'
+import { setUserAuth } from './userServices'
+
+export function getListOfBooks() {
+  setUserAuth()
```

Comment:
This should only be used when authenticating, why are you doing this again in here?